pipeline/src/test_util.rs

```diff
@@ -94,10 +119,6 @@ pub fn test_halo2(_file_name: &str, _inputs: Vec<Bn254Field>) {}
 
 #[cfg(feature = "halo2")]
 pub fn gen_halo2_proof(file_name: &str, inputs: Vec<Bn254Field>) {
-    use powdr_executor::witgen::extract_publics;
-
-    use crate::util::write_or_panic;
```

Comment:
These were moved to the beginning of the file because `gen_estark_proof` also uses it.